CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * Fix: set "java" platform to transactions #1332
 * Feat: Add OkHttp client application interceptor (#1330)
 * Fix: Allow disabling tracing through SentryOptions (#1337)
+* Fix: Fix NPE when MDC contains null values (#1364)
```

Comment:
```suggestion
```

Reply:
@donbeave it should be under `Unreleased`, could you move it? :)

Reply:
@marandaneto Yeah. It's funny how I didn't see it there before. )

Reply:
Fixed.